zaplings/views.py

```diff
@@ -518,31 +507,51 @@ def record_text(request):
     """
     try:
         userid = request.user.pk
+        want_default_text = '_on_'
 
         if userid:
-            # wamt-to-meet
```

Comment:
@dannypernik You've overwritten this function with its earlier version. I had to go in and recover it manually. In the future, I strongly suggest that if any of your changes touch views.py, you do not merge your pull request but leave it open for my review.

Here is the link with your change:
https://github.com/permalife/zaplings-full/commit/8098113f815793ff1a435d3dc45288497818b090#diff-d75b7b863f912cf9e6eb960ca2ab4ad3